readme.md

```diff
@@ -94,6 +94,23 @@ GitHub Enterprise is also supported. More info in the options.
 	</tr>
 </table>
 
+<table>
+	<tr>
+		<th width="50%">
+			Filter pull requests by their build status
+		</th>
+		<th width="50%"></th>
+	</tr>
+	<tr><!-- Prevent zebra stripes --></tr>
+	<tr>
+		<td width="50%">
+			<img src="https://user-images.githubusercontent.com/22439276/56336299-6c961880-61bd-11e9-8dd2-d8884a6f1479.png">
+		</td>
+		<td>
+		</td>
+	</tr>
+</table>
```

Comment:
Sorry for being unclear. I didn't mean to add it here. These are the highlights. This is not a hightlight'able feature.

Reply:
Updated.